clothing_app/clothing_app/select_proxy.py

```diff
@@ -0,0 +1,39 @@
+from clothing_app.observer_weighted_random_Selection import WeightedObservable, WeightedObserver
+
+class ProxyManager(WeightedObservable):
+
+    MAX_WEIGHT = 100
```

Comment:
`MAX_WEIGHT = 100` should be defined in the parent class. Obviously, we can override it here if needed.